lib/iknow_cache.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+require "active_support"
+require "active_support/core_ext/module/delegation"
```

Comment:
Given that we no longer rely on `Rails.logger` and `Rails.cache` but only things with similar interfaces, and configure that up front, do we want/need this dependency outside tests at all?

Reply:
The project is unlikely to be used outside of rails, but `delegate` is such a trivial dependency that it's probably worth removing. 👍